requirements.txt

```diff
@@ -1,5 +1,5 @@
 M2Crypto==0.37.1
-pandas==0.25.1
+pandas==1.3.3
```

Comment:
Were you able to test the update with an exemplary questionnaire response? I would like to prevent conflicts because of version mismatches.